include/rocksdb/options.h

```diff
@@ -428,6 +428,17 @@ struct CompactionServiceJobInfo {
         priority(priority_) {}
 };
 
+struct CompactionServiceScheduleResponse {
+  std::string scheduled_job_id;  // Generated outside of primary host, unique
+                                 // across different DBs and sessions
```

Comment:
"...across different DBs, sessions and jobs"?

Reply:
Yes. Job Id itself within `CompactionServiceJobInfo` is not unique globally.

https://github.com/facebook/rocksdb/blob/99cc36be9bff818e74feb62adb1c1bfe4ce815b6/include/rocksdb/options.h#L414-L417

If users are leveraging centralized queue service externally to offload compactions to remote workers, globally unique identifier is necessary (basically unique across different DBs, sessions and jobs)